### PR TITLE
Merge Release 1 back into master

### PR DESCRIPTION
**Pull request by nambrot**
### Description

Branch `nambrot/master-into-release-1` represents a merge of `master` into `release/celo-core-contracts/1`. The reason why it is separate is to make obvious the merge conflict resolution that happened since we want a clean merge-commit (instead of the typical squash merge). Thus the recommended sequence is:

- [x] Branch `nambrot/master-into-release-1` from `release/celo-core-contracts/1`
- [x] Merge `master` into `nambrot/master-into-release-1` without resolving conflicts
- [x] Resolve conflicts in follow-up commits to make clear how they were resolved
- [x] Open this PR to allow reviewers to review those merge resolutions
- [ ] Upon approval of this PR, close it, update `release/celo-core-contracts/1` to `nambrot/master-into-release-1`
- [ ] Open another PR that represents the merge of `release/celo-core-contracts/1` to `master`

Listed in https://github.com/celo-org/celo-monorepo/issues/4812

The relevant diff to review becomes https://github.com/celo-org/celo-monorepo/pull/5328/files/a675fd719f7fe5ddd676affd69b263f21508b85f..1907dc071c5bf0fe4aaf1e60653523a4bdae66ed (i.e. the commits after the merge commit)

### CODE REVIEW

File: packages/web/src/_page-tests/__snapshots__/about.test.tsx.snap [hunk 1]
```diff
@@ -724,7 +724,7 @@ exports[`About renders 1`] = `
             />
           </audio>
         </div>
-
```
Comment (aaronmgdr) [inline packages/web/src/_page-tests/__snapshots__/about.test.tsx.snap]: someday id like to figure out why there are these whitespace changes sometimes on this files

Comment (nambrot) [reply packages/web/src/_page-tests/__snapshots__/about.test.tsx.snap]: I think it's my VS Code config that automatically delete trailing whitespace (which we really should have everywhere)

File: packages/protocol/contracts/governance/LockedGold.sol [hunk 2]
```diff
@@ -75,9 +77,9 @@ contract LockedGold is
   );
 
   /**
-   * @notice Returns the storage, major, minor, and patch version of the contract.
-   * @return The storage, major, minor, and patch version of the contract.
-   */
+  * @notice Returns the storage, major, minor, and patch version of the contract.
```
Comment (yorhodes) [inline packages/protocol/contracts/governance/LockedGold.sol:80]: this seems errant?

File: packages/protocol/truffle-config.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ const BAKLAVA_FROM = '0x0Cc59Ed03B3e763c02d54D695FFE353055f1502D'
 const BAKLAVASTAGING_FROM = '0x4588ABb84e1BBEFc2BcF4b2296F785fB7AD9F285'
 
 // Gas limit is doubled for initial contract deployment.
-const gasLimit = argv.reset ? 20000000 : 10000000
+const gasLimit = argv.reset ? 20000000 : 12500000
```
Comment (yorhodes) [inline packages/protocol/truffle-config.js:40]: this should be 13M right?

Comment (yorhodes) [reply packages/protocol/truffle-config.js:40]: this can be a followup
I think I had to increase this for the deploy to work

**Review by yorhodes — APPROVED** [review]
hype

**Comment by nambrot** [comment]
Ok with the approval, I will now update `release/celo-core-contracts/1` to `nambrot/master-into-release-1` (da72e7e to 511cd08)

**Pull request merged**